cpp/src/arrow/buffer.h

```diff
@@ -365,7 +374,7 @@ class ARROW_EXPORT TypedBufferBuilder : public BufferBuilder {
   int64_t capacity() const { return capacity_ / sizeof(T); }
 };
 
-/// \brief Allocate a fixed-size mutable buffer from a memory pool
+/// \brief Allocate a fixed size mutable buffer from a memory pool, zero its padding.
```

Comment:
I think "fixed-size" is the correct spelling. Perhaps a native English speaker can confirm?

Reply:
I googled a bit.  "fixed size" seems more common, and reads OK to me